python/ray/dashboard/head.py

```diff
@@ -328,6 +432,13 @@ async def _async_notify():
             namespace=ray_constants.KV_NAMESPACE_DASHBOARD,
         )
 
+        # Parallel start all subprocess modules.
+        for handle in subprocess_module_handles:
+            handle.start_module()
+        # Wait for all subprocess modules to be ready.
+        for handle in subprocess_module_handles:
+            handle.wait_for_module_ready()
+
```

Comment:
We should do this before we start the server like before `_configure_http_server`?